driver/ppm_fillers.c

```diff
@@ -5710,12 +5710,14 @@ int f_sys_dup_e(struct event_filler_arguments *args)
 {
 	int res;
 	unsigned long val;
+	int64_t fd = 0;
```

Comment:
can we use a consistent approach with the rest of the code? 
The pattern we always use is this one:

```
	unsigned long val = bpf_syscall_get_argument(data, 0);
	int32_t fd = (int32_t)val;
	res = bpf_push_s64_to_ring(data, (int64_t)fd);
	CHECK_RES(res);
```

so `fd` as an `int32_t` and than a cast to `int64_t`